src/screens/GroupInfo/index.js

```diff
@@ -310,7 +309,7 @@ const GroupInfo = () => {
 export default GroupInfo;
 
 export const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: '#fff', paddingBottom: 40},
+  container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: 40},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to a constant is a good practice for maintainability and consistency across the application. However, it's important to ensure that `COLORS.white` is indeed equivalent to `'#fff'`. If not, this could lead to unexpected UI changes.
```diff
-  container: {flex: 1, backgroundColor: '#fff', paddingBottom: 40},
+  container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: 40},
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->